src/qibo_cloud_backends/qibo_client.py

```diff
@@ -18,7 +14,7 @@ class QiboClientBackend(NumpyBackend):
         platform (str): Name of the platform. Defaults to `"sim"`.
     """
 
-    def __init__(self, token=None, provider=None, platform=None):
+    def __init__(self, token=None, platform=None):
```

Comment:
For the time being, it's still quite experimental. However, I would consider exposing the `url` in the `QiboClient` object more explicitly (or document it even here) to bolster the *self-hosted* option.